cluster/cluster.go

```diff
@@ -284,3 +297,11 @@ func ConfigureCluster(ctx context.Context, rkeConfig v3.RancherKubernetesEngineC
 
 	return kubeCluster.deployAddons(ctx)
 }
+
+func (c *Cluster) getEtcdProcessHostMap(etcdHosts []*hosts.Host) map[*hosts.Host]v3.Process {
```

Comment:
Can this parameter be more descriptive like `readyEtcdHosts` or you may add a comment explaining the logic of the function, thanks!